libs/api-model-bear/src/visualizationObject/GdcVisualizationObject.ts

```diff
@@ -8,7 +8,15 @@ import isEmpty from "lodash/isEmpty";
 export namespace GdcVisualizationObject {
     export type SortDirection = "asc" | "desc";
     export type Identifier = string;
-    export type MeasureAggregation = "sum" | "count" | "avg" | "min" | "max" | "median" | "runsum";
+    export type MeasureAggregation =
+        | "sum"
+        | "count"
+        | "approximate_count"
```

Comment:
Will the bear backend support this too? Because now it does not: https://github.com/gooddata/gdc-bear/blob/release/resources/specification/md/obj/visualizationObject.res#L100

Reply:
Afaik, it will not.